src/Formula.elm

```diff
@@ -150,6 +266,25 @@ freeTerm t =
     freeTermA t Set.empty
 
 
+
+--not used, + above 2
+
+
+allVariablesInFormula : Formula -> Set String
+allVariablesInFormula f =
```

Comment:
Rozmyslam, ci na nieco treba zoznam vsetkych premennych (v porovnani s vsetkymi volnymi premennymi, co je nizsie). Asi to ma zmysel pre upnost ;)

Nazval by som to mozno `variablesInFormula` a to `freeFormula` nizsie na `freeVariablesInFormula` ? (A potom ten `freeTerm` hore, co je asi `variablesInTerm`).

Druha alternativa je `formulaVariables`, `termVariables`, `freeFormulaVariables` (tie prve dve sa mi pacia, to tretie trochu skripe)

@crnkjck  mas ty nejake preferencie na nazvy? :)

Reply:
Nie, netreba, je to outdated. `allVariablesInFormula` je zmazana. Ale `freeFormula` a `freeTerm` mozem premenovat.